### DISCUSSION TITLE
Share using Context menu in WIndows

### DISCUSSION THREAD

**Discussion by thewaliii** [localsend/localsend#1297] [Discussion (English, International)]
Please consider adding the option to send files through the context menu in Windows or by using the Share function. 
It would be great to have a feature similar to QuickShare and SendAnyWhere, as shown in the attached images.

![2024-04-28_03-55-38](https://github.com/localsend/localsend/assets/48418015/1dbc8599-e449-43be-b3aa-90620cfee54c)
![2024-04-28_03-55-14](https://github.com/localsend/localsend/assets/48418015/c6a0a149-27fa-477e-9810-9214b7e9aafc)

**Comment by Reicoler** [comment]
we need that so much bump!!!
